src/platform/cannonlake/include/platform/shim.h

```diff
@@ -233,6 +233,9 @@
 #define DMWBA_ENABLE		(1 << 0)
 #define DMWBA_READONLY		(1 << 1)
 
+/* DMIC power ON bit */
+#define DMIC_POWER_ON	((uint32_t) BIT(0))
```

Comment:
Please name this after the register and bit, e.g. DMICLCTL_SPA as found in the HAS (otherwise it's difficult to follow). Ditto for the others.

Reply:
@lrgirdwo, I have just applied your suggestion.